string object_name
---
bool object_exists